### DISCUSSION TITLE
Common Data Service has been renamed to Microsoft Dataverse

### DISCUSSION THREAD

**Discussion by JadynWong** [microsoft/PowerPlatform-DataverseServiceClient#74] [Q&A] [answered]
Whether these libraries will change their names? I see that microsoft has renamed "Common Data Service" as "Microsoft Dataverse"

[https://docs.microsoft.com/en-us/powerapps/maker/common-data-service/data-platform-intro](https://docs.microsoft.com/en-us/powerapps/maker/common-data-service/data-platform-intro)

**Comment by MattB-msft** [comment, marked as answer]
Yes,  that is why we have the warning about breaking changes on the first page, 
There will be name changes here and Namespace Updates,  Method names will change as well as property names.. 

We will likely do that in the next few weeks post publishing the Auth Stack changes that are in flight.
